src/data/users.js

```diff
@@ -1,456 +1,611 @@
-//     strideLength: 3.7,
-//     dailyStepGoal: 8000,
-//     friends: [43, 10, 17],
-//   },
-//   {
-//     id: 40,
-//     name: 'Esperanza Schumm',
-//     address: '5719 Hal Fork, Frederikstad SC 23125',
-//     email: 'Otis_Beahan@yahoo.com',
-//     strideLength: 3.5,
-//     dailyStepGoal: 6000,
-//     friends: [29, 28, 3],
-//   },
-//   {
-//     id: 41,
-//     name: 'Meta Leffler',
-//     address: '51598 Sauer Plaza, Port Vicente WY 87161-8441',
-//     email: 'Brice.Greenholt9@hotmail.com',
-//     strideLength: 2.8,
-//     dailyStepGoal: 2000,
-//     friends: [6, 45, 31],
-//   },
-//   {
-//     id: 42,
-//     name: 'Ernestine Heathcote',
-//     address: '9160 Blanche Ford, Dawnmouth OH 11816-1604',
-//     email: 'Brandyn_Quitzon@hotmail.com',
-//     strideLength: 4.5,
-//     dailyStepGoal: 9000,
-//     friends: [20, 25, 22],
-//   },
-//   {
-//     id: 43,
-//     name: 'Alfonso Sporer',
-//     address: '584 Mayert Greens, West Arden SC 97033',
-//     email: 'Jadon_Borer@gmail.com',
-//     strideLength: 4,
-//     dailyStepGoal: 5000,
-//     friends: [2, 19, 43, 33],
-//   },
-//   {
-//     id: 44,
-//     name: 'Cora Rice',
-//     address: '32723 Brekke Burg, West Randallburgh DE 20889',
-//     email: 'Coralie80@yahoo.com',
-//     strideLength: 3.1,
-//     dailyStepGoal: 8000,
-//     friends: [25, 37, 42, 27],
-//   },
-//   {
-//     id: 45,
-//     name: "Jennie O'Hara",
-//     address: '492 Stracke Mews, East Jazlyn OH 58002-5475',
-//     email: 'Elenor.Block12@yahoo.com',
-//     strideLength: 4.4,
-//     dailyStepGoal: 7000,
-//     friends: [35, 39, 31, 25, 33],
-//   },
-//   {
-//     id: 46,
-//     name: 'Vincenzo Hayes',
-//     address: '03146 Bella Ferry, Port Royceberg NE 64057-4380',
-//     email: 'Carlo.Walker@hotmail.com',
-//     strideLength: 3.6,
-//     dailyStepGoal: 4000,
-//     friends: [10, 29, 44],
-//   },
-//   {
-//     id: 47,
-//     name: 'Jevon Koss',
-//     address: '983 Charlotte Island, Schroederchester ME 39894-0208',
-//     email: 'Kasandra_Bashirian@gmail.com',
-//     strideLength: 4.3,
-//     dailyStepGoal: 10000,
-//     friends: [11, 48, 27, 4],
-//   },
-//   {
-//     id: 48,
-//     name: 'Jasper Stracke',
-//     address: '9119 Ziemann Road, Coltberg IA 56671',
-//     email: 'Domenica.Kovacek17@yahoo.com',
-//     strideLength: 3.6,
-//     dailyStepGoal: 7000,
-//     friends: [9, 7, 30],
-//   },
-//   {
-//     id: 49,
-//     name: 'Herbert Douglas',
-//     address: '460 Boyd Viaduct, Florianburgh NM 04038-3499',
-//     email: 'Douglas_Swift99@hotmail.com',
-//     strideLength: 3.8,
-//     dailyStepGoal: 4000,
-//     friends: [29, 15, 32, 37, 31],
-//   },
-//   {
-//     id: 50,
-//     name: 'Jordon Lind',
-//     address: '3866 Jay Loaf, New Felix OR 96784-0274',
-//     email: 'Dane76@hotmail.com',
-//     strideLength: 4.4,
-//     dailyStepGoal: 2000,
-//     friends: [9, 27, 21, 13],
-//   },
-// ];
+export const userData = [
```

Comment:
This is amazing. I think moving everything to a test data file was a great way to go. Let's think about renaming this to testData.js.